gcp/api/server.py

```diff
@@ -1063,8 +1063,13 @@ def query_by_version(context: QueryContext,
           bugs.append(bug)
     elif project == 'oss-vdb-test' and supports_ordering:
       # Query for non-enumerated ecosystems.
-      bugs, next_page_token = yield _query_by_comparing_versions(
-          context, query, ecosystem, version)
+
+      # Performance testing only
+      # TODO(gongh): revert change back after testing.
+      # bugs, next_page_token = yield _query_by_comparing_versions(
+      #     context, query, ecosystem, version)
+      bugs, next_page_token = yield _query_by_generic_version(
+          context, query, package_name, ecosystem, purl, version)
```

Comment:
since these two lines are the same as below, can we just comment out this block?

Reply:
I feel updating here makes it easier to see the difference than commenting out the block. Both do the same thing, and I will revert this change back after the deployment. So I think it doesn't really matter